### PR TITLE
GSoC: Added Thanos project.

### PR DESCRIPTION
**Pull request by bwplotka**
Also added mentor to LFX project.

cc @wiardvanrij @GiedriusS @onprem @squat @matej-g

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Review by GiedriusS — APPROVED** [review]

**Review by squat — APPROVED** [review]

File: lfx-mentorship/2022/01-Spring/README.md [hunk 1]
```diff
@@ -414,7 +414,7 @@ This internship involves improving the existing Katacoda scenario and adding new
 
 - Description: Thanos is a distributed system that has a user interface written in React. Let's create a community instance with continuous integration for easy testing of how Thanos works. Also, it could serve as a testing ground for new React components. A server is provided by CNCF (<https://github.com/cncf/cluster/issues/190>).
 - Recommended Skills: Linux, Ansible, Python, Shell Scripting
-- Mentor(-s): Giedrius Statkevičius (@GiedriusS)
+- Mentor(-s): Giedrius Statkevičius (@GiedriusS), Wiard van Rji (@wiardvanrij)
```
Comment (idvoretskyi) [inline lfx-mentorship/2022/01-Spring/README.md:417]: Please let Wiard to apply as a mentor to the project directly (as suggested here - https://docs.linuxfoundation.org/lfx/mentorship/mentors/create-a-mentor-profile/request-to-be-added-to-a-program).

Comment (bwplotka) [reply lfx-mentorship/2022/01-Spring/README.md:417]: ACK. 

Should I remove this from this PR then?

Comment (idvoretskyi) [reply lfx-mentorship/2022/01-Spring/README.md:417]: @bwplotka no, please keep this in the PR and let them apply to the project.

Comment (wiardvanrij) [reply lfx-mentorship/2022/01-Spring/README.md:417]: Will do, but please fix my name ^^

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Content is good, but please fix @wiardvanrij's name 😊
(https://github.com/cncf/mentoring/pull/502/files#r806257408)

File: lfx-mentorship/2022/01-Spring/README.md (hunk 1 above)
Comment (nate-double-u) [reply lfx-mentorship/2022/01-Spring/README.md:417]: Corrected in [d39ef6e](https://github.com/cncf/mentoring/commit/d39ef6e53c3cd75ae2a2ddb2b3eb0431beef6c61)

Comment (wiardvanrij) [reply lfx-mentorship/2022/01-Spring/README.md:417]: @idvoretskyi  I have applied to the project. Is there something else I need to do? 

Thanks!

Comment (idvoretskyi) [reply lfx-mentorship/2022/01-Spring/README.md:417]: @wiardvanrij I don’t see your application yet.

Comment (idvoretskyi) [reply lfx-mentorship/2022/01-Spring/README.md:417]: @wiardvanrij invited, please let me know is any issues!

**Comment by nate-double-u** [comment]
Hello,

We've been asked to provide some more information about the projects. 

As per the [Defining a Project Idea list](https://google.github.io/gsocguides/mentor/defining-a-project-ideas-list), could you please update any project suggestions you've submitted with the following (if not already provided):

a) a project title/description 
b) more detailed description of the project (2-5+ sentences) 
c) expected outcomes 
d) skills required/preferred 
e) possible mentors 
f) expected size of the project (175 or 350 hours) 
g) an easy, medium, or hard difficulty rating of each project.

Please update your projects by March 3, 2022. Thanks!

**Comment by nate-double-u** [comment]
Please see the umbrella issue: https://github.com/cncf/mentoring/issues/523

Note we have a new deadline: **Tues, March 1 at 5pm PT**.

**Pull request merged**